java/dataflow-connector-examples/src/main/java/com/google/cloud/bigtable/dataflow/example/CsvImport.java

```diff
@@ -81,9 +79,9 @@ public void processElement(DoFn<String, Mutation>.ProcessContext c) throws Excep
         }
 
         Put row = new Put(rowkey);
-        Instant timestamp = c.timestamp();
         for (int i = 1; i < values.length; i++) {
-          row.addColumn(FAMILY, headerBytes[i], timestamp.getMillis(), Bytes.toBytes(values[i]));
+          row.addColumn(FAMILY, headerBytes[i], System.currentTimeMillis(),
```

Comment:
Please create a variable for System.currentTimeMillis() so that all columns have the exact same timestamp.  Other than that, LGTM.

Reply:
Done